integration_test/issue_wizard/ivido/ivido_demo_client.dart

```diff
@@ -0,0 +1,190 @@
+  };
+
+  // There should be a collapsible card for every expected q and a item.
+  final collapsiblesFinder = find.byType(Collapsible);
+  expect(collapsiblesFinder, findsNWidgets(qAndAs.length));
+
+  for (var i = 0; i < qAndAs.entries.length; i++) {
+    final qAndAEntry = qAndAs.entries.elementAt(i);
+    final question = qAndAEntry.key;
+    final answer = qAndAEntry.value;
+
+    // Find the specific collapsible which we are checking this iteration.
+    final collapsibleFinder = collapsiblesFinder.at(i);
+
+    if (!tester.any(collapsibleFinder)) {
+      await tester.scrollUntilVisible(collapsibleFinder, 100);
+    }
+    expect(collapsibleFinder, findsOneWidget);
+
+    // Expect the question text to be present on this collapsible
+    final questionFinder = find.descendant(
+      of: collapsibleFinder,
+      matching: find.text(question).hitTestable(),
+    );
+    expect(questionFinder, findsOneWidget);
+
+    // Unfold answer.
+    await tester.tapAndSettle(questionFinder);
+
+    // The collapsible should have IrmaMarkdown as content
+    final markDownFinder = find.descendant(
+      of: collapsibleFinder,
+      matching: find.byType(IrmaMarkdown),
+    );
+    expect(markDownFinder, findsOneWidget);
+
+    // Because markdown is hard to test we compare the expected markdown with the markdown in the widget
+    // Note: This does not check the actual text on the screen!
+    final markdownWidget = markDownFinder.evaluate().first.widget as IrmaMarkdown;
+    expect(markdownWidget.data, answer);
+
+    // Fold answer again.
+    await tester.tapAndSettle(questionFinder);
+  }
+
+  // Go to the actual issue wizard
+  await tester.tapAndSettle(find.text('Add'));
+
+  // Check the progress indicator
+  final progressIndicatorFinder = find.byType(SessionProgressIndicator);
+  expect(
+    find.descendant(
+      of: progressIndicatorFinder,
+      matching: find.text('Step 1 of 2'),
+    ),
+    findsOneWidget,
+  );
+
+  // Expect stepper with two cards
+  final stepperFinder = find.byType(IrmaStepper);
+  expect(stepperFinder, findsOneWidget);
+
+  final stepperCardsFinder = find.descendant(
+    of: stepperFinder,
+    matching: find.byType(IrmaCard),
+  );
+  expect(stepperCardsFinder, findsNWidgets(2));
+
+  // Expect the right content on the first card
+  expect(
+    tester.getAllText(stepperCardsFinder.first),
+    [
+      'Demo Personal data',
+      'Retrieve your personal data from the dutch personal records database ("Basis Registratie Personen", BRP). You do this with DigiD. You need this information to show who you are.',
+    ],
+  );
+
+  // Expect the right content on the second card
+  expect(
+    tester.getAllText(stepperCardsFinder.at(1)),
+    [
+      'Demo Ivido Login',
+      'Retrieve your Ivido login pass. This way you can log at Ivido in easily and safely.',
```

Comment:
~pass. This way you can log at Ivido in easily and safely~ card to log in to Ivido.